docs/Commands.md

```diff
@@ -0,0 +1,21 @@
+# Commands
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides several vscode commands which are available with `Ctrl+Shift+P`.
+
+![XML Commands](images/Commands/XMLCommands.png)
+
+## Open XML Documentation
+
+This command opens the `XML Documentation`
+
+## Validate current XML file
+
+This command re-triggers the [XML Validation](Validation.md#xml-validation) for the current file.
+
+When the [Server Cache Path](Preferences.md#server-cache-path) is activated, the command removes the referenced XSD, DTD grammar from the local cache.
+
+## Validate current XML file
+
+This command re-triggers the [XML Validation](Validation.md#xml-validation) for the all opened XML files.
+
+When the [Server Cache Path](Preferences.md#server-cache-path) is activated, the command removes all XSD, DTD grammars from the local cache.
```

Comment:
Does it remove all the grammars, or just the grammars referenced by the opened files?

Reply:
all grammars

Reply:
In this case, then I think the command description should same something like "clear remote grammar cache" to make it clear that it clears all the cached grammars.

Reply:
Or "Clear remote grammar cache and revalidate all opened files"

Reply:
fixed